Formula/dub.rb

```diff
@@ -2,10 +2,10 @@ class Dub < Formula
   desc "Build tool for D projects"
   homepage "https://code.dlang.org/about"
   url "https://github.com/dlang/dub.git",
-      :tag => "v1.0.0",
-      :revision => "b59af2b8befb4fad4157d8c9cc86dba707b2fc87"
+      tag: "v1.0.0",
```

Comment:
something weird was going on, i was stuck on homebrew `1.0.1`, all fixed now. Have updated the pr to use hash-rockets.